src/lexers/jsx-lexer.js

```diff
@@ -54,4 +58,59 @@ export default class JsxLexer extends HTMLLexer {
 
     return this.keys
   }
+
+  /**
+   * Recursively convert html tags and js injections to tags with the child index in it
+   *
+   * @param {string} string
+   *
+   * @returns string
+   */
+  eraseTags(string) {
+    const children = this.simplify(acorn.parse(string, {plugins: {jsx: true}}).body[0].expression.children, string);
+
+    const elemsToString = children => children.map((child, index) => {
```

Comment:
Is there a reason you create a function rather than directly returning the mapped children?

Reply:
Yes, it calls itself recurisvely.